spring-integration-core/src/main/java/org/springframework/integration/channel/QueueChannel.java

```diff
@@ -176,11 +179,13 @@ protected boolean doSend(Message<?> message, long timeout) {
 	}
 
 	@Override
+	@ManagedMetric(metricType = MetricType.GAUGE, displayName = "QueueChannel Queue Size")
```

Comment:
These `@ManagedMetric`s are presented on the `QueueChannelManagement` methods. Are they necessary here?
I may guess that ambiguity with `QueueChannelOperations`, but looks like Spring JMX support takes care of that.
Can we confirm this redundancy with some test-case?
Although having them here doesn't hurt...